### PR TITLE
Update SocketModeClient document more; in addition to #1176

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

This pull request updates SocketModeClient documents (in addition to my changes at #1176 ). The document does not have any information about interactivity and slash commands. Also, I've corrected a few errors too.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: docs/_packages/socket_mode.md [hunk 1]
```diff
@@ -90,39 +90,95 @@ const { WebClient } = require('@slack/web-api');
...
 socketModeClient.on('member_joined_channel', async ({event, body, ack}) => {
-    try {
-      // send acknowledgement back to slack over the socketMode websocket connection
-      // this is so slack knows you have received the event and are processing it
-      await ack();
-      await webClient.chat.postMessage({
-          blocks: [
-          {
-            type: 'section',
+  try {
```
Comment (seratch) [inline docs/_packages/socket_mode.md:99]: adjusted indents

File: docs/_packages/socket_mode.md [hunk 2]
```diff
@@ -90,39 +90,95 @@ const { WebClient } = require('@slack/web-api');
...
+
+```javascript
+client.on('slash_commands', async ({ body, ack }) => {
+  if (body.command === "/the-command") {
+    await ack({"text": "I got it!"});
+  }
+});
 ```
+
+For more complex apps, it's worth considering to use Bolt framework. Bolt provides much easier ways to register listeners for events and user actions. Refer to [Bolt's document](https://slack.dev/bolt-js/concepts#socket-mode) for more details.
```
Comment (seratch) [inline docs/_packages/socket_mode.md:180]: Probably, other maintainers can come up with better sentences here. Feel free to add suggestions here.

Comment (mwbrooks) [inline docs/_packages/socket_mode.md:180]:
```suggestion
When your app has multiple interactive events or slash commands, you will need to include your own routing logic. This is a good time to consider using Slack's Bolt framework, which provides an easier ways to register listeners for events and user actions. You can learn more in [Bolt's Socket Mode documentation](https://slack.dev/bolt-js/concepts#socket-mode).
```

**Review by mwbrooks — APPROVED** [review]
Very nice catches!

**Pull request merged**